variables.tf

```diff
@@ -452,6 +452,19 @@ variable "hms_rw_k8s_pdb_settings" {
   }
 }
 
+variable "hms_readwrite_namespace" {
```

Comment:
the variable name is misleading, should we make it more specific ? like `hms_readwrite_metrics_namespace`